### PR TITLE
Add Status and ExecuteType class unit tests #1465

### PR DESCRIPTION
**Pull request by elonlo**
## What is the purpose of the pull request

 - add Status class unit test
 - add ExecuteType class unit test

## Brief change log

  - *Add StatusTest.java*
  - *Add ExecuteType.java*
  - *Optimize Status*
  - *Add UT path to root pom.xml*

## Verify this pull request

This change added tests and can be verified as follows:
  - *Added ExecuteType to verify the change.*
  - *Added StatusTest to verify the change.*
  - *Manually verified the change by testing locally.*

### CODE REVIEW

File: dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/enums/ExecuteTypeTest.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ */
+package org.apache.dolphinscheduler.api.enums;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class ExecuteTypeTest {
+
+    @Test
+    public void getEnum() {
```
Comment (Technoboy-) [inline dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/enums/ExecuteTypeTest.java]: testGetEnum

File: dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/enums/StatusTest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ */
+package org.apache.dolphinscheduler.api.enums;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class StatusTest {
+
+    @Test
+    public void getCode() {
```
Comment (Technoboy-) [inline dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/enums/StatusTest.java]: testGetCode

File: dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/enums/StatusTest.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        assertEquals(Status.SUCCESS.getCode(), 0);
+        assertNotEquals(Status.REQUEST_PARAMS_NOT_VALID_ERROR.getCode(), 0);
+    }
+
+    @Test
+    public void getMsg() {
```
Comment (Technoboy-) [inline dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/enums/StatusTest.java]: testGetMsg

**Comment by elonlo** [comment]
o.o okay

**Pull request merged**